src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+html {
+  --star: url(/src/images/star.svg);
+  --star-activated: url(/src/images/star-active.svg);
+}
+
+body {
+  margin: 0;
+}
+
+.container {
```

Comment:
According to the task conditions, each BEM block should have its separate CSS file. You have two BEM blocks defined, but place all the styles in the single file.